src/main/cpp/DragonVision/DragonLimelight.cpp

```diff
@@ -70,15 +70,16 @@ DragonLimelight::DragonLimelight(
 
 /// @brief Assume that the current pipeline is AprilTag and that a target is detected
 /// @return -1 if the network table cannot be found
-int DragonLimelight::GetAprilTagID()
+std::optional<int> DragonLimelight::GetAprilTagID()
 {
     auto nt = m_networktable.get();
     if (nt != nullptr)
     {
-        return round(nt->GetNumber("tid", -1));
+        int aprilTagInt = int(round(nt->GetNumber("tid", -1)));
```

Comment:
please change to something like:
double value = nt->GetNumber("tid", -1);
int aprilTagInt = static_cast<int>(value + (value > 0 ? 0.5 : -0.5 ))

Reply:
Done